src/Microsoft.Data.SqlClient/netfx/ref/Microsoft.Data.SqlClient.csproj

```diff
@@ -27,5 +27,6 @@
   <ItemGroup>
     <PackageReference Include="Azure.Identity" Version="$(AzureIdentityVersion)" />
     <PackageReference Include="System.Text.Json" Version="$(SystemTextJsonVersion)" />
+    <PackageReference Include="System.Data.Common" Version="$(SystemDataCommonVersion)" />
```

Comment:
I'm kinda confused about this one - it seems like we've needed to build against System.Data.Common for a long time. So I'm concerned why we're only just now adding it. @cheenamalhotra do you know the history behind how we relate to System.Data.Common?

Reply:
It [looks like](https://apisof.net/catalog/5bfef1680a062fbe8a07a16f198d6019?fx=net462) System.Data.Common is packaged in-box with .NET Framework 4.7.1 and upwards, but since we're targeting .NET Framework 4.6.2 we need the package reference. .NET Core doesn't have this problem - all the versions we target package it in-box.

Reply:
I think this should work and shouldn't be a problem, if we also make sure this dependency is added to nuspec. Being OOB allows us to reference types available in NuGet packages for target frameworks where in-box types are missing, like this one.